services/movie.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: List[int] = None,
+               actors_ids: List[int] = None) -> list:
+    films = Movie.objects.all()
+    if genres_ids:
+        films = films.filter(
+            genres__id__in=genres_ids)
+    if actors_ids:
+        films = films.filter(
+            actors__id__in=actors_ids)

```

Comment:
no need to rewrite variable here + fix indentation

Reply:
sorry, didnt understood you clearly. i don't know how to implement solution with no "film = query"